src/components/molecules/AiInsightsTab/AiInsightsTab.tsx

```diff
@@ -0,0 +1,8 @@
+import {usePluginState} from '@plugins/pluginHooks';
+
+const AiInsightsTab = () => {
+  const [aiInsightsProps] = usePluginState<{id: string}>('executionDetailsTabs');
+  return <div>This is a cloud only feature. Please log in to cloud in order to use this feature</div>;
```

Comment:
Maybe:
```suggestion
  return <div>This is a Cloud-only feature. Please [link]connect your cluster to Testkube Cloud[/link] in order to use this feature.</div>;
```

Reply:
test will change with the next PR which contains the actual implementation of the AI Insights. for now, this is a placeholder